policyengine_taxsim/core/output_mapper.py

```diff
@@ -16,7 +16,7 @@ def export_single_household(taxsim_input, policyengine_situation):
         dict: Dictionary of TAXSIM output variables
     """
     mappings = load_variable_mappings()["policyengine_to_taxsim"]
-
+    print(policyengine_situation)
```

Comment:
do we need to keep the print or is it just for internal testing?

Reply:
for internal testing